src/components/message/Message.styles.ts

```diff
@@ -0,0 +1,64 @@
+export const styles = {
+  userInfoStyles: {
+    root: {
+      flexDirection: 'row',
+      gap: '9px'
+    },
+    avatar: {
+      width: '48px',
+      height: '48px'
+    },
+    info: {
+      alignItems: 'baseline',
+      gap: '8px'
+    },
+    myInfo: {
+      flexDirection: 'row-reverse'
+    },
+    interlocutorInfo: {
+      flexDirection: 'row'
```

Comment:
do you really need this styles, because it is 'row' by default?

Reply:
> do you really need this style, because it is 'row' by default?

yup, by default flex-direction of UserProfileInfo is set to column
![зображення](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/9fde34da-6042-443e-b256-a10045a20b1f)